### PR TITLE
Correct the included files to have a working freerdp-nightly-dev package

### PR DESCRIPTION
**Pull request by hardening**

### CODE REVIEW

File: packaging/deb/freerdp-nightly/freerdp-nightly-dev.install [hunk 1]
```diff
@@ -1,4 +1,5 @@
-opt/freerdp-nightly/lib/*.so
```
Comment (bmiklautz) [inline packaging/deb/freerdp-nightly/freerdp-nightly-dev.install]: Without this the development package won't ship the .so files.

File: packaging/deb/freerdp-nightly/freerdp-nightly.install [hunk 2]
```diff
@@ -1,3 +1,3 @@
-opt/freerdp-nightly/lib/*.so.*
```
Comment (bmiklautz) [inline packaging/deb/freerdp-nightly/freerdp-nightly.install]: This was intentional. The library it self is in the freerdp-nigthly package but the .so files (without version), which are only required for compilation, are shipped with the freerdp-nightly-dev. This is the way debian (and also rpm packages) usually do it.

**Comment by bmiklautz** [comment]
@hardening shipping the .a files is ok but I don't agree on the packaging changes for the so files.

**Comment by hardening** [comment]
@bmiklautz I agree, I'm gonna revert the wrong changes

**Comment by bmiklautz** [comment]
@hardening thanks for your efforts. I'll merge it now but I have also created an issue (#2695) that we don't forget to fix the real problem in cmake.

**Comment by bmiklautz** [comment]
@hardening the packages will be built this night and should be available by tomorrow.

**Pull request merged**